VMTouch.cpp

```diff
@@ -0,0 +1,184 @@
+#include "libstuff/libstuff.h"
+#include <cerrno>
+#include <cstddef>
+#include <cstdint>
+#include <fcntl.h>
+#include <linux/fs.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include <sys/resource.h>
+#include <sys/stat.h>
+#include <thread>
+#include <unistd.h>
+
+#include <iostream>
+
+#include <VMTouch.h>
+
+long VMTouch::pagesize = sysconf(_SC_PAGESIZE);
+
+int64_t VMTouch::bytes2pages(int64_t bytes) {
+    int64_t pages = bytes / pagesize;
+    if (bytes % pagesize) {
+        pages++;
+    }
+    return pages;
+}
+
+int VMTouch::aligned_p(void* p) {
```

Comment:
Is this `p` referring to `page`? Let's use that if so

Reply:
I can rename if you want, this function is just copied straight out of the original vmtouch code with the same name.